src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -819,6 +822,7 @@ private void assertTaskDoesNotExist(String caseId, String taskId) {
     }
 
```

Comment:
DelayedTask FT to show warnings are set? Should only create warning when delay has ended and task is created